CRM/Upgrade/Incremental/php/SixTwo.php

```diff
@@ -29,6 +29,38 @@ class CRM_Upgrade_Incremental_php_SixTwo extends CRM_Upgrade_Incremental_Base {
    */
   public function upgrade_6_2_alpha1($rev): void {
     $this->addTask(ts('Upgrade DB to %1: SQL', [1 => $rev]), 'runSql', $rev);
+    $this->addTask('Set upload_date in file table', 'setFileUploadDate');
+    $this->addTask('Set default for upload_date in file table', 'alterSchemaField', 'File', 'upload_date', [
+      'title' => ts('File Upload Date'),
+      'sql_type' => 'datetime',
+      'input_type' => 'Select Date',
+      'required' => TRUE,
+      'readonly' => TRUE,
+      'default' => 'CURRENT_TIMESTAMP',
+      'description' => ts('Date and time that this attachment was uploaded or written to server.'),
```

Comment:
This is kind of interesting. When you install, it didn't used to translate db comments because nobody looks at those. But I suppose these comments are now used in searchkit etc so are already translated. But oddly, even now if you install it still has english db comments even though down below it also has ts(). That might be a timing issue of how ts() works during schema install, or is maybe on purpose. I don't think the db comments need to be translated, so isn't really something that needs fixing, I was more thinking about whether the phrase even needed translating.

But anyway this line in the upgrade now makes it out of sync with a regular install, because it does get translated here. I don't think that's important, just noting.